common-lib/src/cmr/common/api/errors.clj

```diff
@@ -46,12 +49,7 @@
            (csk/->PascalCaseString path-item)))
        field-path))
 
-(defmulti errors->body-string
-  "Converts a set of errors into a string to return in the response body
-  formatted according to the requested response format."
-
-  (fn [response-format errors]
-    response-format))
+;; *************************************
```

Comment:
Why has ;; *******... been added?

Reply:
I want a more physical separation between sets of defmulti calls. error->xml-element and errors->body-string are right on top of each other and I want it clear where they each are.

Reply:
Looking at the full file I now see why.
Can you just convert the three functions into 1 and use a case statement? It will execute much faster that way.  defmulti/defmethods are not needed here.

Reply:
You will see that I did this in other files as well. I also found that in some cases no care was taken to keep the order of things clear, defmethod of different types would be mixed with functions in between them too. I cleaned these up moving the functions outside of the "blocks" and ordering defmulti and defmethods so as to not break load order but also collect them together. The comment line I think may help enforce the seperation with future developers.